sci-biology/bowtie/bowtie-2.2.6.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils toolchain-funcs
+
+DESCRIPTION="Popular short read aligner for Next-generation sequencing data, allowing for gaps"
+HOMEPAGE="http://bowtie-bio.sourceforge.net/bowtie2/"
+SRC_URI="mirror://sourceforge/project/${PN}-bio/${PN}2/${PV}/${PN}2-${PV}-source.zip"
+
+LICENSE="GPL-3"
+SLOT="2"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="examples cpu_flags_x86_sse2 tbb"
+REQUIRED_USE="cpu_flags_x86_sse2"
```

Comment:
Well, I'd say it would be better to use `pkg_pretend()` for this. `REQUIRED_USE` will most likely cause the PM to recommend user enabling this flag which may be undesired. I think it would be better to check for it in `pkg_pretend()` and output appropriate error, to reduce the possibility of accidentally enabling SSE2 when target host doesn't support it.